diff -ur dkimpy-milter-1.2.2.orig/dkimpy_milter/__init__.py dkimpy-milter-1.2.2/dkimpy_milter/__init__.py
--- dkimpy-milter-1.2.2.orig/dkimpy_milter/__init__.py	2020-08-09 20:15:48.000000000 +0200
+++ dkimpy-milter-1.2.2/dkimpy_milter/__init__.py	2022-05-29 13:50:24.310772566 +0200
@@ -451,7 +451,7 @@
 def main():
     # Ugh, but there's no easy way around this.
     global milterconfig
-    configFile = '/usr/local/etc/dkimpy-milter.conf'
+    configFile = '/etc/dkimpy-milter/dkimpy-milter.conf'
     if len(sys.argv) > 1:
         if (sys.argv[1] in ('-?', '--help', '-h')) or len(sys.argv) == 3 or \
                (len(sys.argv) == 4 and sys.argv[2] != '-P'):
@@ -484,7 +484,7 @@
                 else:
                     socketname = 'fd:3'
         if socketname is None:
-            socketname = 'local:/var/run/dkimpy-milter/dkimpy-milter.sock'
+            socketname = 'local:/var/spool/postfix/dkimpy-milter/dkimpy-milter.sock'
     sys.stdout.flush()
     if milterconfig.get('Syslog'):
         syslog.syslog('dkimpy-milter starting:{0} user:{1}'
diff -ur dkimpy-milter-1.2.2.orig/etc/dkimpy-milter.conf dkimpy-milter-1.2.2/etc/dkimpy-milter.conf
--- dkimpy-milter-1.2.2.orig/etc/dkimpy-milter.conf	2020-04-09 04:49:44.000000000 +0200
+++ dkimpy-milter-1.2.2/etc/dkimpy-milter.conf	2022-05-29 13:51:41.948946914 +0200
@@ -8,10 +8,10 @@
 # privileged user (e.g. Postfix)
 UMask			007
 
-# Sign for example.com with key in /usr/local/etc/dkimkeys/dkim.key using
+# Sign for example.com with key in /etc/dkimpy-milter/dkim.key using
 # selector '2007' (e.g. 2007._domainkey.example.com)
 #Domain			example.com
-#KeyFile		/usr/local/etc/mail/dkim.key
+#KeyFile		/etc/dkimpy-milter/example.com.key
 #Selector		default
 
 # Commonly-used options; the commented-out versions show the defaults.
@@ -27,17 +27,18 @@
 # ##  inet:port                   to listen on all interfaces
 # ##  local:/path/to/socket       to listen on a UNIX domain socket
 #
-#Socket local:/run/dkimpy-milter/dkimpy-milter.sock
+#Socket local:/var/spool/postfix/dkimpy-milter/dkimpy-milter.sock
 #
 #Socket			inet:8892@localhost
+Socket local:/var/spool/postfix/dkimpy-milter/dkimpy-milter.sock
 
 ##  PidFile filename
-###      default /run/dkimpy-milter/dkimpy-milter.pid
+###      default /var/run/dkimpy-milter.pid
 ###
 ###  Name of the file where the filter should write its pid before beginning
 ###  normal operations.
 #
-PidFile			/run/dkimpy-milter/dkimpy-milter.pid
+PidFile			/var/run/dkimpy-milter.pid
 
 ##  Userid userid
 ###      default dkimpy-milter
@@ -45,4 +46,4 @@
 ###  Change to user "userid" before starting normal operation?  May include
 ###  a group ID as well, separated from the userid by a colon.
 #
-UserID			dkimpy-milter
+UserID			root:postfix
diff -ur dkimpy-milter-1.2.2.orig/setup.py dkimpy-milter-1.2.2/setup.py
--- dkimpy-milter-1.2.2.orig/setup.py	2020-08-09 20:16:23.000000000 +0200
+++ dkimpy-milter-1.2.2/setup.py	2022-05-29 13:28:25.815906789 +0200
@@ -32,17 +32,17 @@
 class FileMacroExpand(distutils.cmd.Command):
     description = "Expand @@ variables in input files, simlar to make macros."
     user_options = [
-        ('sysconfigdir=', 'e', 'Specify system configuration directory. [/usr/local/etc]'),
-        ('sbindir=', 's', 'Specify system binary directory. [/usr/local/sbin]'),
-        ('bindir=', 'b', 'Specify binary directory. [/usr/loca/bin]'),
-        ('rundir=', 'r', 'Specify run state directory. [/run]'),
+        ('sysconfigdir=', 'e', 'Specify system configuration directory. [/etc]'),
+        ('sbindir=', 's', 'Specify system binary directory. [/usr/sbin]'),
+        ('bindir=', 'b', 'Specify binary directory. [/usr/bin]'),
+        ('rundir=', 'r', 'Specify run state directory. [/var/run]'),
     ]
 
     def initialize_options(self):
-        self.sysconfigdir = '/usr/local/etc'
-        self.sbindir = '/usr/local/sbin'
-        self.bindir = '/usr/local/bin'
-        self.rundir = '/run'
+        self.sysconfigdir = '/etc'
+        self.sbindir = '/usr/sbin'
+        self.bindir = '/usr/bin'
+        self.rundir = '/var/run'
 
     def finalize_options(self):
         self.configdir = self.sysconfigdir + '/dkimpy-milter'
